.travis.d/templates/configmap.yaml.erb

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <%= "#{key}-#{file['owner']}" %>
+  labels:
+    name: <%= "#{key}-#{file['owner']}" %>
+data:
+  path: <%= file['path'] %>
+  contents: '<%= file['contents'].gsub("\n", Regexp.escape("\\n")) %>'
```

Comment:
Use double quotes and put the `!!str` prefix like in other places (in case the file's contents is just a number and it gets converted). Also consider using multi line strings for readability: https://stackoverflow.com/a/21699210